src/jit/lowerarmarch.cpp

```diff
@@ -872,59 +822,19 @@ void Lowering::ContainCheckSIMD(GenTreeSIMD* simdNode)
 //
 void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
```

Comment:
@CarolEidt, @echesakovMSFT 

Just to confirm, there doesn't need to be containment for ARM64 except for immediates, correct?

Reply:
That's my understanding, i.e. there are no Arm64 intrinsics that can take either a register or memory operand.

Reply:
Indeed, not `SIMD` but you do have other intrinsics such as the `prefetch` intrinsics (`pld`) that do take a memory operand. But that's probably out of scope for now I'd imagine.

Reply:
Definitely out of scope for the PR; likely not outside the scope of the total work. We have `Sse.Prefetch0/1/2` and `Sse.PrefetchNonTemporal` on the x86 side already.

Reply:
My point was that we only need to worry about the containment question when the same intrinsic can take *either* a memory or a register operand, which would not be the case for a prefetch.